src/metpy/calc/thermo.py

```diff
@@ -2337,8 +2338,8 @@ def mixed_layer_cape_cin(pressure, temperature, dewpoint, **kwargs):
     of a given upper air profile and mixed-layer parcel path. CIN is integrated between the
     surface and LFC, CAPE is integrated between the LFC and EL (or top of sounding).
     Intersection points of the measured temperature profile and parcel profile are
-    logarithmically interpolated. Kwargs for `mixed_parcel` can be provided, such as `depth`.
-    Default mixed-layer depth is 100 hPa.
+    logarithmically interpolated. Kwargs for `mixed_parcel` can be provided, such as `depth`
+    and `height`. Default mixed-layer depth is 100 hPa.
```

Comment:
From the name of the `height` kwarg it is not clear that this represents an array of height values associated with a vertical profile. Would a variable name of `height_profile` be more appropriate and descriptive for our users? I think this would also help alleviate some confusion between that kwarg and `depth` as they are clearly distinct.